frontends/web/src/locales/en/app.json

```diff
@@ -264,7 +264,12 @@
       "unlock": "Enter BitBox02 password to unlock."
     },
     "stepCreate": {
-      "description": "This name is used as the device name and for the backup.",
+      "description": "This name is used as the device name and for backups.",
```

Comment:
I changed this to get feedback in this review.

This message is currently used when setting the device name before creating a new wallet OR with this pr when restoring from mnemonic.

Phrasing: "the backup" implies that there will be always a sdcard backup. But when restoring from mnemonic, no sdcard is involved. But it is not wrong as the user still could make a backup on a sdcard in the future.

Ok to change it so it works for both create-new-wallet and restore-from-mnemonic ?

let me know then I'll update locize directly.

with this wording:

![Screen Shot 2024-01-26 at 09 19 55](https://github.com/digitalbitbox/bitbox-wallet-app/assets/546900/cdcc377f-051c-4929-a22f-f7ccac6263a8)

Reply:
changed and updated locize.